style.css

```diff
@@ -0,0 +1,147 @@
+/*General*/
+*{
+    margin: 0px;
+
+    padding: 0px;
+    box-sizing: border-box;
+}
+
+body{
+    margin: 0px;
+    padding: 0px;
+}
+img:hover{
+    padding: 5PX;
+}
+.img{
+    transition: padding 15px ease-out;
+}
+
+/*navbar*/
+.navbar{
+    width: auto;
+    height: 11%;
+    justify-items: center;
+}
+
+.form-control{
+    height: 38px;
+    width: 352px;
+    border-radius: none;
+}
+.value{
+    border: none;
+    background: none;
+    margin:0;
+}
+
+.m1{
+    
+    border:none;
+    background: none;
+    margin-left: 195px;
+    position: relative;
+}
+.btn{
+    
+    color: blue;
+    margin-left:-40px;
+    
+}
+.cart{
+    font-size:16px;
+    margin-left:15px;
+    color: whitesmoke;
+    border: none;
+    background:none;
+    
+}
+.nav-link{
+  margin-left: 15px;
+  color: aliceblue;
+ 
+}
+
+.b1{
+border: none;
+width: 100px;
+height: 35px;
+margin-left: 125px;
+float: right;
+color: blue;
+}
+
+/*Deals of the day*/
+.light{
+color: lightslategray;
```

Comment:
Add proper spacing in line no. 76 and 78

Reply:
i have fixed it